src/adv.py

```diff
@@ -62,17 +62,22 @@ def inv_item(item):
 
 
 def move_player(direction, current_location):
-    attribute = direction + '_to'
-    if hasattr(current_location, attribute):
-        return getattr(current_location, attribute)
+    attribute = f'{direction}_to'
+    if hasattr(room[current_location], attribute):
```

Comment:
I think the way you're handling the room change is behind the invalid input error. You might want to consider moving this method to the player class.